cli/quicktype.ts

```diff
@@ -100,7 +120,32 @@ const sections = [
   }
 ];
 
+function optionDefinitionsForRenderer(renderer: Renderer): OptionDefinition[] {
+  return _.map(renderer.options, o => {
+    return {
+      name: o.name,
+      description: o.description,
+      typeLabel: o.typeLabel,
+      renderer: true,
+      type: String as any } as OptionDefinition;
+    });
+}
```

Comment:
Is there any reason we might want to prefix renderer options by name? For example, if the C# renderer specifies an option `use-lists`, might we want to present it as `--csharp-use-lists`?

Reply:
It's verbose but would it be worthwhile to make the option parsing simpler?

Reply:
Maybe, maybe not.  I don't like it.